RELEASE_NOTES

```diff
@@ -1,3 +1,5 @@
+- Improvement: FENIX-15817 - Add support to setup credentials in CgdStudentPhotoClient constructor
```

Comment:
@ajsco Release notes are not in qubIT's standard. The correct form is:

```
- Improvement: Add support to setup credentials in CgdStudentPhotoClient constructor
Comment [FENIX-15817]
```

Reply:
Só não percebi se é literalmente
Comment [FENIX-15817]
ou 
[FENIX-15817]

Reply:
Já fiz novo commit&push

Reply:
Erro meu :facepalm: é apenas [Fenix-15817]